packages/react-native/ReactCommon/react/renderer/components/textinput/platform/ios/react/renderer/components/iostextinput/TextInputShadowNode.cpp

```diff
@@ -140,6 +140,32 @@ Size TextInputShadowNode::measureContent(
       .size;
 }
 
+Float TextInputShadowNode::baseline(
+    const LayoutContext& layoutContext,
+    Size size) const {
+  auto attributedString = getAttributedString(layoutContext);
+
+  if (attributedString.isEmpty()) {
+    auto placeholder = getConcreteProps().placeholder;
+    auto string = !placeholder.empty()
+        ? placeholder
+        : BaseTextShadowNode::getEmptyPlaceholder();
+    auto textAttributes = getConcreteProps().getEffectiveTextAttributes(
+      layoutContext.fontSizeMultiplier);
+    attributedString.appendFragment({string, textAttributes, {}});
+  }
+  
+  // I don't think I should be reading directly from yogaNode, but leyout metrics
```

Comment:
```suggestion
  // I don't think I should be reading directly from yogaNode, but layout metrics
```

Reply:
Replaced the comment in https://github.com/facebook/react-native/pull/45102/commits/b7be70090ade7947288312fadcd689b06485b508